### PR TITLE
Implemented Two getCheck and runCheck tests

### PR DESCRIPTION
**Pull request by Alien1993**

### CODE REVIEW

File: test/python/datadog_checks/base/checks/__init__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+
+class __AgentCheckPy2(object):
+    def __init__(self, *args, **kwargs):
+        pass
+
+    def run(self):
+        return "result"
+
+    @classmethod
```
Comment (masci) [inline test/python/datadog_checks/base/checks/__init__.py]: staticmethod, there was an error in the original check being addressed here https://github.com/DataDog/integrations-core/pull/3127

File: test/python/fake_agent/fake.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from datadog_checks.base.checks import AgentCheck
+
+# Fake agent for testing purposes
+class FakeAgent(AgentCheck):
```
Comment (masci) [inline test/python/fake_agent/fake.py]: nit: can we rename it to `FakeCheck`? Technically this is a check

File: test/python/datadog_checks/base/checks/__init__.py (hunk 1 above)
Comment (Alien1993) [reply test/python/datadog_checks/base/checks/__init__.py]: Sure!

File: test/python/fake_agent/fake.py (hunk 2 above)
Comment (Alien1993) [reply test/python/fake_agent/fake.py]: :+1:

Comment (Alien1993) [reply test/python/fake_agent/fake.py]: Do I rename the module too?

Comment (masci) [reply test/python/fake_agent/fake.py]: ah yes please

**Pull request merged**